app/config/app.php

```diff
@@ -169,14 +169,14 @@
      *   The conventional location for custom renderers is in `src/Error`. Your exception renderer needs to
      *   implement the `render()` method and return either a string or Http\Response.
      *   `errorRenderer` - string - The class responsible for rendering PHP errors. The selected
-     *   class will be used for both web and CLI contexts. If you want different classes for each environment 
+     *   class will be used for both web and CLI contexts. If you want different classes for each environment
```

Comment:
I just wonder what's the cause of removing the space at the end of the word. Is it because of the IntelliJ IDEA? Please kindly clarify.

Reply:
it was intentional because is my configuration in vs code I could see a dots like this:
![image](https://github.com/ArielGalvez/tutorial-cakephp/assets/32244920/a3d5aa3c-3751-4a5a-b3cf-5e9335e0858e)